vmdb/app/controllers/api_controller.rb

```diff
@@ -95,6 +96,13 @@ def initialize
     @api_token_mgr   = TokenManager.new(@module)
   end
 
+  def self.all_encrypted_attributes
+    (ATTR_TYPES[:encrypted]             |
+     ::Authentication.encrypted_columns |
+     ::MiqDatabase.encrypted_columns    |
```

Comment:
I do need my list for the normalizer, considering I can be rendering regular hashes like configurations, provisioning requests options, etc and not just objects.  One option is to update my list dynamically upon rendering db objects.  Let me think about it further.